ass2-task/sass/pages/_common.scss

```diff
@@ -0,0 +1,153 @@
+body,
+html {
+  font-family: 'Roboto-Regular', sans-serif;
+  margin: 0;
+  height: 100%;
+  overflow-y: hidden;
+  width: 100%;
+}
+
+* {
+  box-sizing: border-box;
+  font-smoothing: antialiased;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+span,
+ul,
+li,
+a,
+button,
+input,
+select,
+textarea,
+table tr td {
+  margin: 0;
+  padding: 0;
+}
+
+ul li {
+  font-weight: 200;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+h1 {font-size: 24px; font-weight: 500;}
+h2 {font-size: 22px; font-weight: 400;}
+h3 {font-size: 20px; font-weight: 400;}
+h4 {font-size: 18px; font-weight: 300;}
+h5 {font-size: 16px; font-weight: 300;}
+h6 {font-size: 14px; font-weight: 200;}
+p,
+ul li,
+a,
+button,
+input,
+label,
+select,
+textarea,
+table tr td {
+  font-size: 16px;
+}
+
+button {
+  cursor: pointer;
+}
+
+input {
+  border: 0;
+  outline: none;
+}
+
+.clear {
+  clear: both;
+}
+
+.left {
+  float: left;
+}
+
+.right {
+  float: right;
+}
+
+.red {
+  color: $red;
+}
+
+.orange {
```

Comment:
@priyanka0901 You can generate the background color and text color class dynamically by just passing the color variables. Check [this](https://github.com/Infratab/FTMobile/pull/479) how to do